docs/src/main/sphinx/connector/object-storage-file-formats.rst

```diff
@@ -0,0 +1,112 @@
+===========================
+Object storage file formats
```

Comment:
Add a section for each format that at least says what it is and has an anchor for the header .. and then potentially more config stuff